pkg/verify/verify.go

```diff
@@ -47,25 +58,37 @@ var validReleaseDigest = regexp.MustCompile(`^[a-zA-Z0-9:]+$`)
 // digest - all verifiers must have at least one valid signature attesting the release
 // digest. If any failure occurs the caller should assume the content is unverified.
 type ReleaseVerifier struct {
-	verifiers     map[string]openpgp.EntityList
-	stores        []*url.URL
+	verifiers map[string]openpgp.EntityList
+
+	locations     []*url.URL
 	clientBuilder ClientBuilder
+	stores        []SignatureStore
 
 	lock           sync.Mutex
 	signatureCache map[string][][]byte
 }
 
 // NewReleaseVerifier creates a release verifier for the provided inputs.
-func NewReleaseVerifier(verifiers map[string]openpgp.EntityList, stores []*url.URL, clientBuilder ClientBuilder) *ReleaseVerifier {
+func NewReleaseVerifier(verifiers map[string]openpgp.EntityList, locations []*url.URL, clientBuilder ClientBuilder) *ReleaseVerifier {
 	return &ReleaseVerifier{
 		verifiers:     verifiers,
-		stores:        stores,
+		locations:     locations,
 		clientBuilder: clientBuilder,
 
 		signatureCache: make(map[string][][]byte),
 	}
 }
 
+// WithStores copies the provided verifier and adds any provided stores to the list.
+func (v *ReleaseVerifier) WithStores(stores []SignatureStore) *ReleaseVerifier {
```

Comment:
`stores ...SignatureStore` so folks adding a single store don't need to build a slice?